tests/posix/eventfd/testcase.yaml

```diff
@@ -5,5 +5,7 @@ common:
     - eventfd
   integration_platforms:
     - qemu_x86
+  platform_exclude:
+    - qemu_cortex_a53_smp
```

Comment:
maybe add a comment linking to a GH issue?

Reply:
done